main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/OrlandoRomo/academy-go-q32021/infrastructure/api"
+	"github.com/OrlandoRomo/academy-go-q32021/infrastructure/router"
+	"github.com/OrlandoRomo/academy-go-q32021/registry"
+)
+
+func main() {
+	var UrbanDictionaryApiKey = flag.String("urban-dictionary-api-key", envString("URBAN_DICTIONARY_API_KEY", "55a4103f9amsh99bbd454b45f012p180fb6jsnf682ba32da43"), "Urban Dictionary api key")
```

Comment:
Don't insert sensitive data as api-keys into the repo, you could probably get this info from a config file